config/software/designs.json

```diff
@@ -909,6 +909,22 @@
       "curvedSeam"
     ]
   },
+  "umbra": {
+    "code": "Joost De Cock, Jonathan Haas",
+    "description": "A FreeSewing pattern for a basic, highly-customizable underwear pattern",
+    "design": "Joost De Cock, Jonathan Haas",
```

Comment:
Do you want to be credited here? I have no problem leaving that in, but I'm not sure you want to be credited here, especially considering that you just took over Uma yourself.

Reply:
I don't mind, but I also don't want people to think they have to credit me just because they are re-using something I wrote/designed.

So feel free to take my name off if you want.